lib/diaspora_federation/entities/account_rename.rb

```diff
@@ -0,0 +1,25 @@
+module DiasporaFederation
+  module Entities
+    # This entity is issued when a person changes their diaspora ID (e.g. when a user migration
+    # from one to another pod happens).
+    #
+    # @see Validators::AccountRenameValidator
+    class AccountRename < Entity
+      # @!attribute [r] author
+      #   The old diaspora* ID of the person who changes their ID
+      #   @see Person#author
+      #   @return [String] author diaspora* ID
+      property :author, :string
+
+      # @!attribute [r] person
+      #   Holds new updated data of a person, including diaspora ID, public key, etc
+      #   @return [Person] person new data
+      entity :person, Entities::Person
+
+      # @return [String] string representation of this object
+      def to_s
+        "AccountRename:#{author}"
```

Comment:
I would add the new diaspora ID here too: `"AccountRename:#{author}:#{person.author}"`